sdk/netapp/azure-resourcemanager-netapp/CHANGELOG.md

```diff
@@ -1,14 +1,76 @@
 # Release History
 
-## 1.7.0-beta.1 (Unreleased)
+## 1.7.0 (2025-04-17)
 
-### Features Added
+- Azure Resource Manager NetAppFiles client library for Java. This package contains Microsoft Azure SDK for NetAppFiles Management SDK. Microsoft NetApp Files Azure Resource Provider specification. Package tag package-2025-01-01. For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
 
 ### Breaking Changes
 
-### Bugs Fixed
+#### `models.Volume$Definition` was modified
+
+* `withIsRestoring(java.lang.Boolean)` was removed
```

Comment:
Breaking change due to swagger fix:
https://github.com/Azure/azure-rest-api-specs/pull/33247#issuecomment-2730713901

The property is never settable.